pom.xml

```diff
@@ -367,54 +367,58 @@
                 <version>${maven-license-plugin.version}</version>
                 <inherited>true</inherited> <!-- for multi-module config -->
                 <configuration>
-                    <header>${license.header}</header>
                     <aggregate>true</aggregate> <!-- for multi-module config -->
                     <skipExistingHeaders>false</skipExistingHeaders>
-                    <includes>
-                        <!-- Add files where to include licensing -->
-                        <include>src/**/*.java</include>
-                        <include>src/**/*.xml</include>
-                        <include>src/**/*.proto</include>
-                        <include>**/*.css</include>
-                        <include>**/*.sh</include>
-                    </includes>
-                    <excludes>
-                        <!-- Add files to exclude from licensing -->
+                    <!-- Add comment style for protocol buffers -->
+                    <mapping>
+                        <java>SLASHSTAR_STYLE</java>
+                        <proto>DOUBLESLASH_STYLE</proto>
+                    </mapping>
+                    <licenseSets>
+                        <licenseSet>
+                            <header>${LicenseSet.header}</header>
+                            <includes>
+                                <!-- Add files where to include licensing -->
+                                <include>src/**/*.java</include>
+                                <include>src/**/*.xml</include>
+                                <include>src/**/*.proto</include>
+                                <include>**/*.css</include>
+                                <include>**/*.sh</include>
+                            </includes>
+                            <excludes>
+                                <!-- Add files to exclude from licensing -->
 
-                        <!-- Exclude pom -->
-                        <exclude>**/pom.xml</exclude>
+                                <!-- Exclude pom -->
+                                <exclude>**/pom.xml</exclude>
 
-                        <!-- Exclude config and scripts -->
-                        <exclude>**/dev-utils/*</exclude>
-                        <exclude>dev-support/artifacts/**</exclude>
-                        <exclude>dev-support/work/**</exclude>
+                                <!-- Exclude config and scripts -->
+                                <exclude>**/dev-utils/*</exclude>
+                                <exclude>dev-support/artifacts/**</exclude>
```

Comment:
Can you check all of these ?
Looks like this was copied from HBase or Phoenix.
We should clean up the non-existent directories and files 
(except for the ones generated during build)